spec/models/user_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
+
```

Comment:
i know the user spec is probably beyond the scope of this PR, but be sure to add one later!  :)